x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/setup_page/index.tsx

```diff
@@ -14,11 +15,39 @@ import {
   EuiTitle,
   EuiSpacer,
   EuiIcon,
+  EuiCallOut,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiCode,
+  EuiCodeBlock,
+  EuiLink,
 } from '@elastic/eui';
 import { useCore, sendPostFleetSetup } from '../../../hooks';
 import { WithoutHeaderLayout } from '../../../layouts';
 import { GetFleetStatusResponse } from '../../../types';
 
+export const RequirementItem: React.FunctionComponent<{ isMissing: boolean }> = ({
+  isMissing,
+  children,
+}) => {
+  return (
+    <EuiFlexGroup gutterSize="s" alignItems="flexStart">
+      <EuiFlexItem grow={false}>
+        <EuiText>
+          {isMissing ? (
+            <EuiIcon type="crossInACircleFilled" color="danger" />
```

Comment:
This code looks fine but I want to confirm that the `isMissing` state actually works. Your screenshot shows all the requirements as passing, which shouldn't be possible, because if all requirements are met, the page goes away.

Reply:
I harcoded a test for debugging. Just updated the screenshot with running ES without API keys